process.py

```diff
@@ -1125,14 +1066,12 @@ def build(release,package):
        firefox_version = value.strip()
     if key == 'errata_url':
        errata_url_base = value.strip()
-    if key == 'bugzilla_url':
-       bugzilla_url_base = value.strip()
-    if key == 'bugzilla_login':
-       bugzilla_login = value.strip()
-    if key == 'bugzilla_api_key':
-       bugzilla_api_key = value.strip()
     if key == 'centos_fork':
        centos_fork = value.strip()
+    if key == 'jira_url':
+       jira_url_base = value.strip()
+    if key == 'jira_api_key':
+       jira_api_key = value.strip()
```

Comment:
We should add the jira_url_base and jira_api_key to the sample config file and remove the bugzilla versions.